requirements.txt

```diff
@@ -0,0 +1,2 @@
+Flask==0.10.1
+lxml==3.4.2
```

Comment:
I added the dependency in requirements.txt and created mock-requirements.txt without gpio because it will not be executed in a non-production environment.